package/MDAnalysis/analysis/rms.py

```diff
@@ -516,8 +516,8 @@ def __init__(self, atomgroup, reference=None, select='all',
         #   *groupselections* groups each a dict with reference/mobile
         self._groupselections_atoms = [
             {
-                'reference': self.reference.universe.select_atoms(*s['reference']),
-                'mobile': self.atomgroup.universe.select_atoms(*s['mobile']),
+                'reference': self.reference.select_atoms(*s['reference']),
+                'mobile': self.atomgroup.select_atoms(*s['mobile']),
```

Comment:
I reviewed the codebase and thought It's not essential and wrong to call .universe, so I just removed the character .universe.

Reply:
Going by the docstring, this seems to have been an intentional design decision in order to provide an atom selection that goes beyond the passed atomgroups.

I'll let someone with more familiarity with that feature speak up though.

Reply:
Yes -- this is handy, for example, if you want to align on C-alphas and then calculate the RMSD of backbone atoms.